transactional_licence_form/forms.py

```diff
@@ -62,69 +60,72 @@ def __init__(self, *args, **kwargs):
         super(OrganizationForm, self).__init__(*args, **kwargs)
         self.initial["agent_country"] = "country:GB"
 
-    title = "About your organisastion"
+    title = "Step 3 - About your organisastion"
 
     agent_companyname = fields.FCLCharField(
-        label="3. What is the full legal name of your organisation?", max_length=50
+        label="4. What is the full legal name of your organisation?", max_length=50
```

Comment:
Since the PR mentioned text field lengths, I got thinking:
"LOCUS INTERNATIONAL CENTRE FOR ENTREPRENEURSHIP DEVELOPMENT AND INCUBATION SERVICES LIMITED" is an outlier at 83 characters, and https://find-and-update.company-information.service.gov.uk/company/04120480/filing-history is 160... :)

Reply:
Bumped it up to 100, but might be worth going for 200 just in case.